packages/ui/src/apps/collectionsUi/viewModules/start/Component.js

```diff
@@ -1,126 +1,128 @@
-                Login
-                <Icon name="right arrow" />
-              </Button>
-            </NavLink>
-          </Container>
-        </Segment>
-
-        <Segment style={{ padding: '8em 0em' }} vertical>
-          <Grid container stackable verticalAlign="top">
-            <Grid.Row>
-              <Grid.Column width={8}>
-                <Segment padded="very">
-                  <Header as="h3" style={{ fontSize: '2em' }}>
-                    Collection management for large installations
-                  </Header>
-                  <p style={{ fontSize: '1.33em' }}>
-                    {`At the core of the system is support for assembling, managing
-                  and sharing data associated with natural history collections
-                  and their curation ("collection management"). Target
-                  collections include zoological, botanical, geological and
-                  paleontological collections, living collections, biodiversity
-                  inventories, observation records, and molecular data. DINA is
-                  primarily intended for large installations servicing the
-                  collection management needs of a country, a region, or a large
-                  institution.`}
-                  </p>
-                </Segment>
-                <Divider horizontal />
-                <Segment padded="very">
-                  <Header as="h3" style={{ fontSize: '2em' }}>
-                    An international partnership for open-source development
-                  </Header>
-                  <p style={{ fontSize: '1.33em' }}>
-                    {`DINA is developed by the DINA consortium, an unincorporated
-                  international partnership among organizations and individuals
-                  for collaborative open-source development. The DINA consortium
-                  was founded in 2014 by six natural history collection
-                  institutions in Europe and North America and is open to
-                  additional members as detailed below. The DINA acronym stands
-                  for "DIgital Information system for NAtural history data", and
-                  has its roots in a Swedish initiative to replace a
-                  heterogeneous collection of unsustainable in-house databases
-                  with a modern, web-based national collection management
-                  system.`}
-                  </p>
-                </Segment>
-              </Grid.Column>
-              <Grid.Column floated="right" width={6}>
-                <Segment padded="very">
-                  <Markdown
-                    fallbackLanguage="en"
-                    textKey="modules.start.changelog"
-                  />
-                </Segment>
-              </Grid.Column>
-            </Grid.Row>
+        <div className="ui center aligned grid">
+          <Grid
+            columns={2}
+            style={{ marginTop: '6.25em', maxWidth: '60em' }}
```

Comment:
if you want a maxWidth like this it's better to add the prop `container` that achieves the same thing.

You should look at those:
https://react.semantic-ui.com/collections/grid/#responsive-variations-container
https://react.semantic-ui.com/elements/container/

In general it would be good if you look more into the Semantic docs so you use the built-in features when possible